--- configure.in.gnome2	2010-04-15 04:00:40.000000000 +0200
+++ configure.in	2010-04-15 04:03:11.000000000 +0200
@@ -186,7 +186,7 @@
 	AC_CHECK_PROG(PKG_TEST,pkg-config,yes)
 	if test $PKG_TEST; then
 		#GNOME >= 2
-		GPILOTCONF=`gnome-config --libdir`"/pkgconfig/gnome-pilot-2.0.pc"
+		GPILOTCONF="$libdir/pkgconfig/gnome-pilot-2.0.pc"
 	        if !(test -f "$GPILOTCONF"); then
 			#But no gnome-pilot-2
 			AC_MSG_RESULT([Gnome 2 with gnome-pilot < 2])
@@ -203,10 +203,10 @@
 		GPILOTCONF=`gnome-config --libdir`"/gpilotConf.sh"
 		GPILOTCONFPROG="gnome-config"
 		GPILOTVERSION="gpilot"
-	fi
 	if !(test -f "$GPILOTCONF"); then
 		AC_MSG_ERROR([*** file $GPILOTCONF does not exist. Please install gpilot-devel files ***])
 	fi
+	fi
 	GPILOT=yes
 	AC_SUBST(GPILOT)
 	AC_SUBST(GPILOTCONFPROG)
